cli/src/main/resources/help/server.adoc

```diff
@@ -51,6 +53,9 @@ EXAMPLES
 `server report` +
 Obtains a server report, including information about network, threads, memory, etc.
 
+`server heap-dump` +
+Generates a JVM heap dump in the server data directory, returning the path of the generated file.
```

Comment:
I am getting weird feelings of posting the actual path of the file. This seems to be exposing too much info to the user, letting them know the folder layout. I would suggest just returning the name of the file which can be found in the server's data directory.

Reply:
I have changed it so that it just returns the filename. Bear in mind that only ADMIN members can invoke this operation.